#!/usr/bin/env python
PACKAGE = "blaser_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("brightness_min", int_t, 0, "Lower threshold of laser brightness", 70, 0, 255)
gen.add("hue_min", int_t, 0, "Lower threshold of hue. For red, use min>max", 118, 0, 180)
gen.add("hue_max", int_t, 0, "Upper threshold of hue. For red, use min>max", 124, 0, 180)

exit(gen.generate(PACKAGE, "blaser_ros", "LaserDetector"))